CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this extension will be documented in this file.
 
 ## Unreleased
 
+- Add new actions in the context (right-click) menu for Clusters & Topics to generate a new project.
```

Comment:
Minor adjustment for clarity so folks don't expect it to be available for SR and/or upcoming Flink compute pools (yet)
```suggestion
- Add new actions in the context (right-click) menu for Kafka clusters & Topics to generate a new project.
```